test/Dialect/Torch/decompose-complex-ops.mlir

```diff
@@ -735,6 +735,20 @@ func.func @torch.aten.index_put(%input: !torch.vtensor<[?],f32>, %index: !torch.
   return %0 : !torch.vtensor<[?],f32>
 }
 
+// -----
+// CHECK-LABEL:   func @torch.aten.index_add(
+// CHECK-SAME:                              %[[INP:.*]]: !torch.vtensor<[?],f32>, %[[INDEX:.*]]: !torch.vtensor<[?],f32>,
+// CHECK-SAME:                              %[[VALUES:.*]]: !torch.vtensor<[?],f32>, -> !torch.vtensor<[?],f32> {
+// CHECK:           %[[INDEX:.*]] = torch.vtensor<[?]>,f32 %[[INDEX]]
+// CHECK:           %[[RES:.*]] = torch.valsem.aten.index_put_impl %[[INP]], %[[INDEX]], %[[VALUES]]] : !torch.vtensor<[?],f32>, !torch.vtensor<[?],f32>, !torch.vtensor<[?],f32> -> !torch.vtensor<[?],f32>
+// CHECK:           return %[[RES]] : !torch.vtensor<[?],f32>
+
+
+func @torch.aten.index_add(%input: !torch.vtensor<[?],f32>, %dim: !torch.int, %index: !torch.vtensor<[?],f32>, %values: !torch.vtensor<[?],f32>, -> !torch.vtensor<[?],f32> {
+  %0 = torch.aten.index_add %input, %dim, %index, %values, : !torch.vtensor<[?],f32>, !torch.int, !torch.vtensor<[?],f32>, !torch.vtensor<[?],f32> -> !torch.vtensor<[?],f32>
```

Comment:
I think this is invalid MLIR. There should not be a comma after the last argument to `torch.aten.index_add`. Also, there `alpha` argument is missing

Reply:
@lorenzbc Could you confirm this?